intake_stac/catalog.py

```diff
@@ -333,6 +333,15 @@ def stack_bands(self, bands, path_as_pattern=None, concat_dim='band'):
 
         return CombinedAssets(configDict)
 
+    def _yaml(self):
+        data = {'metadata': {}, 'sources': {}}
+        data['metadata'].update(self.metadata)
+        for key, source in self.items():
+            data['sources'][key] = source._yaml()['sources']['stac_asset']
```

Comment:
Effectively, you recreate the stac object from this string unique identifier? If I've understood it correctly, then everything seems in order here.

Reply:
Yes exactly